### PR TITLE
Add glance tempest tests to ceph-backends.yml

### PR DESCRIPTION
**Pull request by maximsava12**
As a pull request owner and reviewers, I checked that:
- [ ] Appropriate testing is done and actually running
- [ ] Appropriate documentation exists and/or is up-to-date:
  - [ ] README in the role
  - [ ] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Comment by maximsava12** [comment]
@dependabot rebase

File: scenarios/centos-9/ceph_backends.yml [hunk 1]
```diff
@@ -32,4 +32,14 @@ cifmw_tempest_default_groups: &test_operator_list
...
 cifmw_tempest_default_jobs: *test_operator_list
+cifmw_tempest_tempestconf_profile:
+  create: true
+  debug: true
+  os_cloud: default
+  overrides:
+    image.image_path: "http://download.cirros-cloud.net/0.5.1/cirros-0.5.1-x86_64-disk.img"
+    image.http_image: "http://download.cirros-cloud.net/0.5.1/cirros-0.5.1-x86_64-disk.img"
+    image-feature-enabled.import_image: true
+    image-feature-enabled.api_v2: true
```
Comment (tosky) [inline scenarios/centos-9/ceph_backends.yml:45]: isn't this enabled by default?

File: scenarios/centos-9/ceph_backends.yml [hunk 2]
```diff
@@ -32,4 +32,14 @@ cifmw_tempest_default_groups: &test_operator_list
...
   - neutron-operator
   - nova-operator
+  - glance-operator
 cifmw_tempest_default_jobs: *test_operator_list
+cifmw_tempest_tempestconf_profile:
+  create: true
+  debug: true
+  os_cloud: default
+  overrides:
+    image.image_path: "http://download.cirros-cloud.net/0.5.1/cirros-0.5.1-x86_64-disk.img"
```
Comment (tosky) [inline scenarios/centos-9/ceph_backends.yml:42]: isn't this set by default by tempestconf?

File: scenarios/centos-9/ceph_backends.yml [hunk 3]
```diff
@@ -32,4 +32,14 @@ cifmw_tempest_default_groups: &test_operator_list
...
+    image.http_image: "http://download.cirros-cloud.net/0.5.1/cirros-0.5.1-x86_64-disk.img"
```
Comment (tosky) [inline scenarios/centos-9/ceph_backends.yml:43]: isn't this set by default by tempestconf?

**Comment by cjeanner** [comment]
@maximsava12 dependabot doesn't rebase patches that aren't owned by dependabot.

Otherwise - is this patch still needed? Do you need any help to finalize it?

**Comment by cjeanner** [comment]
@maximsava12 hello! Is this patch still needed and active? If not, care to close it and, once you have appropriate time for it, propose a new version if needed?

**Comment by pablintino** [comment]
Closed due to inactivity. Feel free to reopen it if needed.

**Pull request closed without merging**